third_party/3/pkg_resources/__init__.pyi

```diff
@@ -205,15 +205,18 @@ class DistributionNotFound(ResolutionError): ...
 
 class VersionConflict(ResolutionError):
     @property
-    def dist(self) -> List[Any]: ...
+    def dist(self) -> Any: ...
     @property
-    def req(self) -> List[Any]: ...
+    def req(self) -> Any: ...
     def report(self) -> str: ...
-    def with_context(self, required_by: Dict[str, Any]) -> ContextualVersionConflict: ...
+    if bool(required_by) == False:
+        def with_context(self, required_by: Dict[str, Any]) -> VersionConflict: ...
+    else:
+        def with_context(self, required_by: Dict[str, Any]) -> ContextualVersionConflict: ...
```

Comment:
This will not work. There is currently no way to express that `required_by` is an empty dict.

Reply:
Ah! Reverting and just letting it return `ContextualVersionConflict` then.